### DISCUSSION TITLE
#234とおなじです。トラックボールの動作が安定しないです。

### DISCUSSION THREAD

**Discussion by unagiin** [Yowkees/keyball#238] [Q&A]
カーソルを少し動かした時の挙動が安定しません。カーソルの動きが急に速くなってしまう時があります。
#234 の方と同じ現象だとおもいます。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
はんだ付け時に光学センサーが破損したでしょうか？ちょっと経験のない症状です。

ファームウェアを書き直しても解決しないでしょうか？

はんだ付けの見直しやファーム書き換えで解決しないようでしたら、お手数ですが解決方法を相談したいと思いますので、白銀ラボのお問合せからメッセージをいただけますでしょうか。
https://shirogane-lab.com/pages/contact
